packages/react-radio/src/components/RadioGroup/RadioGroup.test.tsx

```diff
@@ -11,8 +12,12 @@ describe('RadioGroup', () => {
 
   // TODO add more tests here, and create visual regression tests in /apps/vr-tests
 
+  beforeEach(() => {
+    resetIdsForTests();
+  });
+
   it('renders a default state', () => {
-    const result = render(<RadioGroup>Default RadioGroup</RadioGroup>);
+    const result = render(<RadioGroup />);
     expect(result.container).toMatchSnapshot();
```

Comment:
What do you think about eliminating snapshots in unit tests as a general rule? This could be changed to 
```tsx
const { getByRole } = render(<RadioGroup/>);
expect(getByRole('radiogroup').classList.includes('fui-RadioGroup')).toBeTrue();
```

Reply:
I agree about removing snapshots. I plan to update tests in a future PR. I'll leave this as-is for now, and follow up with another PR that adds tests and removes these snapshots.